test/library/standard/CommDiagnostics/commDiagsTable.comm-none.good

```diff
@@ -0,0 +1,40 @@
+| locale |
+| -----: |
+|      0 |
+
+| locale |
+| -----: |
+|      0 |
+
+| locale |
+| -----: |
+|      0 |
+
+| locale |
+| -----: |
+|      0 |
+
+| locale |
+| -----: |
+|      0 |
+
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |
+
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |
+
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |
+
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |
+
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |
```

Comment:
Why print out `unstable`? To me `commDiagsPrintUnstable` says print out the comm counts even for unstable values, so it seems odd to me to print `unstable` instead of the value.

Reply:
Sounds like I got my boolean logic backwards, not for the first time.